File: 2.0/java/campaigns/getCampaignSound.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.CampaignSound;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        Long id = 379506003L;
        // use second argument to limit returned fields
        CampaignSound campaignSound = client.campaignSoundsApi().get(id, "name,lengthInSeconds");
    }
}

File: 2.0/csharp/campaigns/getCampaignSound.cs
using CallfireApiClient;
using CallfireApiClient.Api.Campaigns.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        // limit returned fields if needed, e.g. "name,status,lengthInSeconds"
        CampaignSound campaignSound = client.CampaignSoundsApi.Get(379506003, "name,lengthInSeconds");
    }
}
